.github/slack-notifiaction.sh

```
@@ -0,0 +1,39 @@
+author=${author/frcroth/<@felix.roth>}
+author=${author/dieknolle3333/<@charlie.meister>}
+channel="webknossos-bots"
+commitmsg=$(git log --format=%s -n 1)
+pullregex="(.*)#([0-9]+)(.*)"
+
+while [[ "$commitmsg" =~ $pullregex ]]; do
+    commitmsg="${BASH_REMATCH[1]}#<https://github.com/scalableminds/webknossos/issues/${BASH_REMATCH[2]}|${BASH_REMATCH[2]}>${BASH_REMATCH[3]}"
+done
+
+buildlink="<https://circleci.com/gh/scalableminds/webknossos/${}|ready>"
+mesg="${author} WEBKNOSSOS docker image \`master__${GITHUB_RUN_ID}\` for “${commitmsg}” is ${buildlink}."
```

Shellcheck (0.10.0)

[warning] This is a unicode quote. Delete and retype it (or ignore/singlequote for literal).

(SC1111)